# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../scribus/glibc234.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


A trival fix for compilation with gcc-2.3.4 ...

  - Rene Rebe <rene@exactcode.de>

--- scribus-1.1.6/scribus/page.cpp.orig	2004-05-15 23:16:48.585516928 +0200
+++ scribus-1.1.6/scribus/page.cpp	2004-05-15 23:17:39.016850200 +0200
@@ -15,6 +15,8 @@
  *                                                                         *
  ***************************************************************************/
 
+#include <unistd.h> // unlink
+
 #include "page.h"
 #include "page.moc"
 #include "pageitem.h"
